src/components/Crypto/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {CryptoTypes} from '../../types';
+import {
+  CryptoContainer,
+  NameContainer,
+  Price,
+  Logo,
+  Increase,
+  Decrease,
+  Name,
+  Slug,
+  Arrow,
+  Arrowdown,
+  Arrowup,
+  Percentage,
+} from '../Crypto/styles';
+
+const Crypto = ({name, slug, price, market_porcentage, image}: CryptoTypes) => {
+  return (
+    <CryptoContainer>
+      <Logo source={image} />
+      <NameContainer>
+        <Name>{name}</Name>
+        <Slug>{slug}</Slug>
+      </NameContainer>
+      <Price>${price}</Price>
+      {parseInt(market_porcentage) > 0.0 ? (
+        <Percentage>
+          <Increase>{market_porcentage}%</Increase>
+        </Percentage>
+      ) : (
+        <Percentage>
+          <Decrease>{market_porcentage}%</Decrease>
+        </Percentage>
+      )}
+      {parseInt(market_porcentage) > 0.0 ? (
+        <Arrow>
+          <Arrowup source={require('../../assets/Arrowup.png')} />
```

Comment:
Don't use require to import the imgs, use import:
```
import ArrowDown from "../../assets/ArrowDown.png"

<Arrow source={ArrowDown}
```
also rename the image file name...

And the component Arrowup and Arrowdown should be the same component, the arrows are identical and you are using imgs

Reply:
So, adapt the validation to this case